DOKS-wordpress/README.md

```diff
@@ -150,6 +151,143 @@ Finally, you need to setup the trusted sources between your MySQL database and y
+
+Below command lists the available storage classes for your Kubernetes cluster:
+
+```console
+kubectl get sc
+```
+
+The output looks similar to:
+
+```text
+NAME                         PROVISIONER                 RECLAIMPOLICY   VOLUMEBINDINGMODE   ALLOWVOLUMEEXPANSION   AGE
+do-block-storage (default)   dobs.csi.digitalocean.com   Delete          Immediate           true                   24h
+```
+
+DigitalOcean Block Storage Volumes are mounted as read-write by a single node (RWO). Additional nodes cannot mount the same volume. The data content of a PersistentVolume can not be accessed by multiple Pods simultaneously.
+
+Horizontal pod autoscaling (HPA) policies are used to scale the WordPress Pods in a dynamically StatefulSet.
+
+This is why WordPress requires a [volume](https://kubernetes.io/docs/concepts/storage/volumes/) that can be mounted as read-write by many nodes (RWX), in your case, WordPress replica Pods.
+
+NFS (Network File System) is a commonly used solution to provide RWX volumes on block storage. This server offers a PersistentVolumeClaim (PVC) in RWX mode so that multiple web applications can access the data in a shared fashion.
+
+OpenEBS Dynamic NFS Provisioner allows users to create an NFS PV that sets up a new Kernel NFS instance for each PV on top of the user's choice of backend storage.
+
+**Note:**
+
+Please visit [OpenEBS](https://openebs.io/) for more details.
+
+Next, you will install the OpenEBS Dynamic NFS Provisioner on your Kubernetes cluster using the [OpenEBS Helm Chart](https://github.com/openebs/dynamic-nfs-provisioner). For your purposes you will be installing and configuring only the dynamic nfs provisioner since that's what you need.
+
+First, clone the `container-blueprints` repository and change directory to your local copy and the `DOKS-wordpress` folder in it:
+
+```shell
+
+git clone https://github.com/digitalocean/container-blueprints.git
+cd container-blueprints/DOKS-wordpress
+
+```
+
+Next, add the `Helm` repository:
+
+```console
+helm repo add openebs-nfs https://openebs.github.io/dynamic-nfs-provisioner
+
+helm repo update
+```
+
+Next, open and inspect the `assets/manifests/openEBS-nfs-provisioner.yaml` file provided in the repository:
+
+```yaml
+nfsStorageClass:
+  backendStorageClass: "do-block-storage"
+```
+
+**Note:**
+
+The override is self-explanatory and can be customized. Please visit [openebs nfs provisioner helm values](https://github.com/openebs/dynamic-nfs-provisioner/blob/develop/deploy/helm/charts/values.yaml) for the full values.yaml file and more details.
```

Comment:
Please avoid as much as possible constructs like: `self explanatory`, `it's obvious that`, `it's very simple to do something`, etc. Main point here is that readers may not be so knowledgeable as you are. So, in general even if some things `are obvious`, we should give a brief explanation for the reader.